easyrest/models/user.py

```diff
@@ -33,6 +33,7 @@ class User(Base):
     birth_date = Column(Date)
     password = Column(Text)
     role_id = Column(Integer, ForeignKey('user_roles.id'), default=1)
+    is_active = Column(Integer, default=0)
```

Comment:
Probably [Boolean](https://docs.sqlalchemy.org/en/latest/core/type_basics.html#sqlalchemy.types.Boolean) is better here.